package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "glue",
   "description": "Server composer for hapi.js",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
I don't think it should be a patch release, this doesn't make sense for people with standard npm semvers.